test/elixir_auth_google_test.exs

```diff
@@ -165,6 +169,29 @@ defmodule ElixirAuthGoogleTest do
              "https://foobar.com/auth/google/callback"
   end
 
+  test "generate_oauth_url(url) passing in App.Endpoint.url() #94" do
+    url = "gcal.fly.dev"
+    client_id = ElixirAuthGoogle.google_client_id()
```

Comment:
see https://github.com/dwyl/elixir-auth-google/pull/95/files#r1197717102
I would just check the url contains the information we want (https, redirect_uri, ... query parameters) instead of checking the all url